packages/jest-config/src/Deprecated.ts

```diff
@@ -28,6 +28,11 @@ const deprecatedOptions: DeprecatedOptions = {
 
   Please update your configuration.`,
 
+  init: () =>
```

Comment:
Currently this config doesn't work, the behavior will be fixed in separate PR as described here https://github.com/jestjs/jest/pull/14490#issuecomment-1710038662

Reply:
Resolved